index.html

```diff
@@ -0,0 +1,24 @@
+<!-- @format -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Document</title>
```

Comment:
- [ ] Kindly include a more descriptive title for your page. It will be okay if you can make it a habit as it is also a best practice that enhances how Search Engines rank your page and increase access to your content. :wink: - `Document` is too vague and generic. How about something like `Hello page` or something similar that suggests what the page is about?